.github/actions/release-notes/action.yml

```diff
@@ -0,0 +1,30 @@
+name: Release Notes
```

Comment:
This code is moved here from the release-notes workflow.

Note that the tag name is taken explicitly as an input, rather than from ref_name which might have an unexpected value, as we have seen.